mysqld_exporter.go

```diff
@@ -656,6 +656,9 @@ type Exporter struct {
 	totalScrapes    prometheus.Counter
 }
 
+// Suffix to use with SHOW SLAVE STATUS
+var slaveStatusSuffix string
```

Comment:
I would put this higher up and not between the `Exporter` definition and its constructor.